astropy/coordinates/baseframe.py

```diff
@@ -359,6 +361,59 @@ def _insert(self, obj, values, axis=0):
         return out
 
 
+class CoordinateSharedMethods:
+    @property
+    def masked(self):
+        """Whether the underlying data is masked.
+
+        Raises
+        ------
+        ValueError
+            If the frame has no associated data.
+        """
+        return self.data.masked
+
+    def get_mask(self, *attrs):
+        """Get the mask associated with these coordinates.
+
+        Parameters
+        ----------
+        *attrs : str
+            Attributes from which to get the masks to combine. Items can be
+            dotted, like ``"data.lon", "data.lat"``. By default, get the
+            combined mask of all components (including from differentials).
```

Comment:
The docstring should mention that frame attributes can accessed too, but they are not accessed by default.